### DISCUSSION TITLE
Plot Manager Failed with Error No 12 "Not Enough Space"

### DISCUSSION THREAD

**Discussion by Imexglobal222** [swar/Swar-Chia-Plot-Manager#461] [Q&A]
Can someone please help me with this issue. My Plot Manager kept on failing with errors message below saying that I don't have enough Space. When the Space on C: (OS), Temp, and Dest drive still have a lot of space left. This thing kept on showing up every 4-5 hours after it running. Thanks
![image](https://user-images.githubusercontent.com/48275934/118422828-cd340000-b678-11eb-837b-b2ae1d62b7fb.png)

**Comment by vantantech** [comment]
I have the same issue .. it's the viewer so I just ignore it and restart the viewer

**Comment by mshwinfo** [comment]
I have the same issue
